index.html

```diff
@@ -0,0 +1,107 @@
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+        <ul>
+            <li><img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"/></li>
+      
+          <li><a class="nav-link" href="#Features">Features</a></li>
+          <li><a class ="nav-link" href="#How-It-Works">How It Works</a></li>
+          <li><a class ="nav-link" href ="#Pricing">Pricing</a></li>
+        </ul>
+        </nav>
+      </header>
+      
+
+        <section id="Features">
+            <div class="grid">
+                <div class="icon">
+                    <i class="fa-solid fa-fire"></i>
+                </div>
+                <div class="desc">
+                    <h2>Premium Materials</h2>
+                    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+                </div>
+            </div>
+            <div class="grid">
+                <div class="icon">
+                    <i class="fa-fire"></i>
+                </div>
+                <div class="desc">
+                    <h2>Fast Shipping</h2>
+                    <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. </p>
+                </div>
+            </div>
+            <div class="grid">
+                <div class="icon">
+                    <i class="fa-solid fa-fire"></i>
+                </div>
+                <div class="desc">
+                    <h2>Quality Assurance</h2>
+                    <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+                </div>
+            </div>
+        </section>
+        <section id="How-It-Works">
+            <iframe id="video" src ="https://www.youtube.com/watch?v=y8Yv4pnO7qc" allowfullscreen="" height="250" frameboarder="0"></iframe>
+        </section>
+        <section id="Pricing">
+            <div id="tenor" class="product">
+                <div class="level">Tenor Trombone</div>
+                <h2>$600</h2>
+                <ol>
+                    <li>Lorem, ipsum.</li>
+                    <li>Lorem, ipsum.</li>
+                    <li>Lorem, ipsum dolor.</li>
+                    <li>Lorem, ipsum.</li>
+                </ol>
+                <button class="btn">Select</button>
+            </div>
+           
+
+            <div id="bass" class="product">
+                <div class="level">Bass Trombone</div>
+                <h2>$900</h2>
+                <ol>
+                    <li>Lorem, ipsum.</li>
+                    <li>Lorem, ipsum.</li>
+                    <li>Lorem, ipsum dolor.</li>
+                    <li>Lorem, ipsum.</li>
+                </ol>
```

Comment:
Please do not leave generated text in the code

Reply:
Your comment looks descriptive and polite but let's make it better by adding a reason😉 don't forget, the reason may be obvious to you but not to the user😉 kindly explain why you are requesting the changes😉